src/module/handlebars.js

```diff
@@ -84,6 +84,10 @@ function registerHandlebarsHelpers() {
     return (a / b) * 100;
   });
 
+  Handlebars.registerHelper('getProperty', function (obj, key) {
```

Comment:
This is redundant with Handlebars **built-in** helper  `lookup`
https://handlebarsjs.com/guide/builtin-helpers.html#lookup

e.g:
```hbs
{{lookup obj key}}
```